packages/app/components/Settings/SettingsPage.tsx

```diff
@@ -141,7 +141,11 @@ export default function SettingsPage({
             <Upload
               customRequest={({ file }) => handleUpload(file)} // Use customRequest to handle the upload logic ourselves
               beforeUpload={beforeUpload}
-              showUploadList={true}
+              showUploadList={false}
```

Comment:
Make sure it's matching master branch, these new lines shouldn't be added

Reply:
I don't know why this change is here. I did not do this.

Reply:
can you sync this page with main?